metal3-dev/local-cbo.sh

```diff
@@ -18,6 +33,11 @@ fi
 cd $cbo_path
 
 export RUN_NAMESPACE=openshift-machine-api
-oc apply -f manifests/
+oc apply -f manifests/0000_31_cluster-baremetal-operator_01_images.configmap.yaml
+oc apply -f manifests/0000_31_cluster-baremetal-operator_02_metal3provisioning.crd.yaml
+oc apply -f manifests/0000_31_cluster-baremetal-operator_03_baremetalhost.crd.yaml
+oc apply -f manifests/0000_31_cluster-baremetal-operator_04_serviceaccount.yaml
+oc apply -f manifests/0000_31_cluster-baremetal-operator_05_rbac.yaml
+oc apply -f manifests/0000_31_cluster-baremetal-operator_07_clusteroperator.cr.yaml
```

Comment:
This will discard the state already managed by the CBO in its CO, and I don't think we want to do that.

Reply:
@dhellmann thanks for review. I understand your concerns about these commands. As far as I understand, your concerns are also valid for the previous version `oc apply -f manifests/`.

Reply:
Yes, good point. We probably shouldn't have been doing that either. :-)

Reply:
Initially such approach was required as a way also to test the generated manifests (during a local development or CI). I think it could be reviewed now (maybe to be more granular), considering both the latest developments done on CBO and the fact our CI is slightly more robust on this side - thanks to the upgrade jobs (but maybe in a different PR)

Reply:
@andfasano @dhellmann now I remove `oc apply` to prevent possible incompatibilities in this PR and will work on granular changes in different PR. What do you think?